gaffer-as-a-service/gaas-jwt-sidecar/src/main/java/uk/gov/gchq/gaffer/gaas/sidecar/controller/CustomFilter.java

```diff
@@ -59,9 +62,12 @@ public Mono<Void> filter(final ServerWebExchange exchange, final GatewayFilterCh
         // JWT Token is in the form "Bearer token". Remove bearer word and get
         // only the token
         if (requestTokenHeader != null && requestTokenHeader.startsWith("Bearer ")) {
+            logger.info("JWT Token is in the form \"Bearer token\" = ");
             jwtToken = requestTokenHeader.substring(7);
+            logger.info("Remove bearer word and get only the token = ");
             try {
                 username = jwtTokenUtil.getUsernameFromToken(jwtToken);
+                logger.info("Get username from the token = ");
```

Comment:
Why is there an `=` at the end of this line?

Reply:
Changed as per the review comments